### ISSUE TITLE
POST problem with version 2.8.6

### ISSUE THREAD

**Issue by jozefrebjak** [synackray/vcenter-netbox-sync#115]
### Environment
* Python version:  3.6.9
* NetBox version: 2.8.6
* vCenter version: 6.50
* vCenter NetBox Sync version:  latest

### Steps to Reproduce

There is an issue with latest version of netbox. 

I am getting error with creating an object

```
[CRITICAL] Well this in unexpected. Please report this. POST request received 500 status with body
```

```
The complete exception is provided below:

<class 'TypeError'>

__call__() missing 1 required positional argument: 'serializer_field'
```

**Comment by jozefrebjak** [comment]
Ok I found where was a problem. Problem is not related with `vcenter-netbox-sync`, but with install of Netbox.

**Comment by fjasboka** [comment]
Would you mind posting what caused it in case other people run in to it. Could it be the systemd service file not pointing to the netbox venv that solved it?

**Comment by jozefrebjak** [comment]
@fjasboka  I ran legacy `supervisor` method, So I migrate to `systemd`. I used default one from `/contrib`. I just had some problems to get starting netbox with `systemd` there was only problem with `--config /opt/netbox/gunicorn.py netbox.wsgi` from `ExecStart` because I had a `gunicorn_config.py` instead of `gunicorn.py` in my `/opt/netbox/` directory. I'm running latest 2.8.6 version of netbox where it's specified in ExecStart to use `gunicorn` from `/opt/netbox/venv/bin/` folder.
